ios/keyman/Keyman/Keyman-Info.plist

```diff
@@ -58,14 +79,37 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>ITSAppUsesNonExemptEncryption</key>
-	<false/>
 	<key>UISupportedInterfaceOrientations~ipad</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>UTExportedTypeDeclarations</key>
+	<array>
+		<dict>
+			<key>LSItemContentTypes</key>
+			<array>
+				<string>com.pkware.zip-archive</string>
+			</array>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.data</string>
+			</array>
+			<key>UTTypeDescription</key>
+			<string>Keyman Package</string>
+			<key>UTTypeIdentifier</key>
+			<string>com.keyman.kmp</string>
```

Comment:
Does this tie to the app's product ID/name? Is that along the lines of `com.tavultesoft.keyman...`?

Reply:
Where possible we should try and use `com.keyman` and not `com.tavultesoft` for the future as we own the keyman.com domain but not the tavultesoft.com domain.

Reply:
@darcywong00 this is actually not related to the identifier of the app itself, but it is just creating an ID for a custom file extension.